### PR TITLE
Fix wrong when statement preventing key transfer

### LINKED ISSUE #46: Key not read by servers that need it
Hi Brian,

Thanks for your work on this role!

When adding a server after initial setup, the key is not correctly read by the server.

I think it's a wrong check;
https://github.com/brianshumate/ansible-consul/blob/master/tasks/main.yml#L125

```yaml
- name: Read key for servers that require it
  set_fact:
    consul_raw_key: "{{ lookup('file', '/tmp/consul_raw.key') }}"
  when: consul_raw_key is not defined and bootstrap_marker.stat.exists
```
Servers that require it don't have the `bootstrap_marker` file. So I think the check should be `... and not bootstrap_marker.stat.exists`.

Happy to provide a PR if you can confirm.

### PR DESCRIPTION
**Pull request by groggemans**
Also introduces list notation for longer when statements. This makes the
statements more clear and prevents future logic errors.

Fixes #46

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -112,22 +126,31 @@
...
+    - consul_raw_key is defined
+    - bootstrap_marker.stat.exists
 
 - name: Read key for servers that require it
   set_fact:
     consul_raw_key: "{{ lookup('file', '/tmp/consul_raw.key') }}"
-  when: consul_raw_key is not defined and bootstrap_marker.stat.exists
+  when:
+    - consul_raw_key is not defined
+    - not bootstrap_marker.stat.exists
```
Comment (groggemans) [inline tasks/main.yml:146]: Added the `not` here to fix the bug.

Should have done it in a separate commit, but thought too late about it.

**Comment by groggemans** [comment]
@brianshumate I'm running into an issue with further testing. Best to wait with merging until I figure it out.

**Comment by brianshumate** [comment]
Thanks again!

**Comment by groggemans** [comment]
@brianshumate Looks like you missed the comment.

It fixed setting up a server after initial setup, but broke the initial setup. Looks like a bigger logic change is necessary here to fix the issue for all possible situations.

I'm looking into it. You may wan't to revert the PR in the mean time. I hope to have a fix later this evening.
